api_app/tests_ma/test_api/test_routes/test_airlock.py

```diff
@@ -96,14 +96,16 @@ async def test_post_airlock_request_with_non_deployed_workspace_id_returns_404(s
         response = await client.post(app.url_path_for(strings.API_CREATE_AIRLOCK_REQUEST, workspace_id=WORKSPACE_ID), json=sample_airlock_request_input_data)
         assert response.status_code == status.HTTP_404_NOT_FOUND
 
+    @patch("services.aad_authentication.AzureADAuthorization.get_workspace_role_assignment_details", return_value={"researcher_emails": ["researcher@outlook.com"], "owner_emails": ["owner@outlook.com"]})
```

Comment:
Instead of adding the patch every line, can we put it in the log_in_with_researcher_user func under the "with patch" section? line 75